### DISCUSSION TITLE
can't load http://localhost:10000/builder successfully

### DISCUSSION THREAD

**Discussion by Yueming-Yan** [Budibase/budibase#14092] [Help]
Hi there,

I got 502 when I tried to load http://localhost:10000/builder
![image](https://github.com/Budibase/budibase/assets/67954596/c3c713e0-5f7b-400a-b184-30737119ab09)

these are containers after I ran 'yarn dev', I guess they are less than expected？
![image](https://github.com/Budibase/budibase/assets/67954596/d1c1c5a6-b2d9-4417-bc47-e590fda7bf3b)

then I checked the log of 'yarn dev', there are some fishy printouts I guess may cause this issue, could you please help to check? Thanks a lot!!!

**ONE**
>Lerna (powered by Nx)   No access token found
Nx will continue running, but nothing will be written or read from the remote cache.
Run details will also not be available in the Nx Cloud UI.

**TWO**
>@budibase/worker: node:internal/fs/utils:348
@budibase/worker:     throw err;
@budibase/worker:     ^
@budibase/worker: Error: ENOENT: no such file or directory, open '/home/-/budibase/packages/worker/dist/passwordRecovery.hbs'
@budibase/worker:     at Object.openSync (node:fs:590:3)
@budibase/worker:     at Object.openSync (/home/-/budibase/node_modules/dd-trace/packages/datadog-instrumentations/src/fs.js:265:57)
@budibase/worker:     at readFileSync (node:fs:458:35)
@budibase/worker:     at readFileSync (/home/-/budibase/node_modules/dd-trace/packages/datadog-instrumentations/src/fs.js:265:57)
@budibase/worker:     at readStaticFile (/home/-/budibase/packages/worker/dist/index.js:1582:37)
@budibase/worker:     at src/constants/templates/index.ts (/home/-/budibase/packages/worker/dist/index.js:1650:54)
@budibase/worker:     at __init (/home/-/budibase/packages/worker/dist/index.js:9:57)
@budibase/worker:     at src/utilities/email.ts (/home/-/budibase/packages/worker/dist/index.js:15628:5)
@budibase/worker:     at __init (/home/-/budibase/packages/worker/dist/index.js:9:57)
@budibase/worker:     at src/sdk/users/users.ts (/home/-/budibase/packages/worker/dist/index.js:15694:5)
@budibase/worker:     at __init (/home/-/budibase/packages/worker/dist/index.js:9:57)
@budibase/worker:     at src/sdk/users/index.ts (/home/-/budibase/packages/worker/dist/index.js:15710:5)
@budibase/worker:     at __init (/home/-/budibase/packages/worker/dist/index.js:9:57)
@budibase/worker:     at Object.<anonymous> (/home/-/budibase/packages/worker/dist/index.js:24108:1)
@budibase/worker:     at Module._compile (node:internal/modules/cjs/loader:1267:14)
@budibase/worker:     at Module._extensions..js (node:internal/modules/cjs/loader:1321:10) {
@budibase/worker:   errno: -2,
@budibase/worker:   syscall: 'open',
@budibase/worker:   code: 'ENOENT',
@budibase/worker:   path: '/home/-/budibase/packages/worker/dist/passwordRecovery.hbs'
@budibase/worker: }
@budibase/worker: Node.js v20.0.0
@budibase/worker: [nodemon] **app crashed** - waiting for file changes before starting...

**Comment by ConorWebb96** [comment]
If you aren't able to get it up and running via the codebase. I'd recommend trying to use Budibase via the CLI instead. Its a much easier way to get started with Budibase you can read up on this [here](https://docs.budibase.com/docs/budibase-cli-setup).

**Reply by Yueming-Yan** [reply]
Thank you for your suggestion
